pages/sign-up/signup.css

```diff
@@ -0,0 +1,130 @@
+.grids-wrapper>*{
+    color:black;
+    /* background-color: var(--greyLight-1); */
+    box-sizing: border-box;
+}
+
+.rows3-grid{
+    display: grid;
+    height: 100vh;
+    grid-gap: 2rem;
+    grid-template-rows: 8rem 1fr 8rem;
+    grid-template-areas:
+        "row-grid1 "
+        "row-grid2"
+        "row-grid3";  
+}
+
+.header {
+    grid-area: row-grid1;
+    color: var(--white);
+    background-color:var(--pink8bg);
+}
+
+main{
+    grid-area: row-grid2;
+    background-color:var(--pink8bg);
+    padding: 3rem 0;
+    display: flex;
+    justify-content: center;
+
+}
+
+footer {
+    grid-area: row-grid3;
+    justify-content: center;
+    background-color:var(--pink8bg);
+}
+
+.tab-wrapper{
+    height: 100%;
+    flex-basis: 40rem;
+    background-color: var(--pink8);
+    color: var(--gold1);
+    box-shadow:  5px 5px 12px #120f1d,
+    -1px -1px 12px #141120;
+    
+}
```

Comment:
here var color should be better

Reply:
you mean to put box-shadow in var. okeiy i missed maybe i did for some ,will do.